
Loading components for TIF035

Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 30, in <module>
    energy = bulk.get_potential_energy()
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    return self._calc.get_potential_energy(self)
    energy = bulk.get_potential_energy()
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    self.calculate(atoms, converge=True)
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
      File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
energy = bulk.get_potential_energy()
    return self._calc.get_potential_energy(self)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    return self._calc.get_potential_energy(self)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    self.calculate(atoms, converge=True)
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    self.calculate(atoms, converge=True)
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 230, in calculate
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    self.set_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    self.wfs.set_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    self.set_positions(atoms)
    WaveFunctions.set_positions(self, spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 307, in set_positions
    spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    self.wfs.set_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
    spos_ac = self.initialize_positions(atoms)
    WaveFunctions.set_positions(self, spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
    self.wfs.set_positions(spos_ac)
    self.wfs.set_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    self.wfs.set_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
    WaveFunctions.set_positions(self, spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
    WaveFunctions.set_positions(self, spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
        WaveFunctions.set_positions(self, spos_ac)
spos_ac = self.initialize_positions(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 299, in initialize_positions
    self.wfs.set_positions(spos_ac)
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    raise ValueError('Some atom is too close to the zero-boundary!')
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
ValueError: Some atom is too close to the zero-boundary!
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
GPAW CLEANUP (node 6): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
    self.wfs.set_positions(spos_ac)
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
    raise ValueError('Some atom is too close to the zero-boundary!')
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
ValueError: Some atom is too close to the zero-boundary!
    WaveFunctions.set_positions(self, spos_ac)
GPAW CLEANUP (node 0): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
        WaveFunctions.set_positions(self, spos_ac)
self.wfs.set_positions(spos_ac)
    raise ValueError('Some atom is too close to the zero-boundary!')
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/fdpw.py", line 33, in set_positions
ValueError:     raise ValueError('Some atom is too close to the zero-boundary!')
Some atom is too close to the zero-boundary!
GPAW CLEANUP (node 5): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
ValueError: Some atom is too close to the zero-boundary!
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
GPAW CLEANUP (node 3): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
    raise ValueError('Some atom is too close to the zero-boundary!')
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
ValueError: Some atom is too close to the zero-boundary!
    raise ValueError('Some atom is too close to the zero-boundary!')
GPAW CLEANUP (node 7): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
ValueError: Some atom is too close to the zero-boundary!
GPAW CLEANUP (node 2): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
    WaveFunctions.set_positions(self, spos_ac)
    raise ValueError('Some atom is too close to the zero-boundary!')
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/wavefunctions/base.py", line 192, in set_positions
ValueError: Some atom is too close to the zero-boundary!
GPAW CLEANUP (node 1): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
    rank_a = self.gd.get_ranks_from_positions(spos_ac)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/domain.py", line 103, in get_ranks_from_positions
    raise ValueError('Some atom is too close to the zero-boundary!')
ValueError: Some atom is too close to the zero-boundary!
GPAW CLEANUP (node 4): <type 'exceptions.ValueError'> occurred.  Calling MPI_Abort!
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 6 in communicator MPI_COMM_WORLD 
with errorcode 42.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun has exited due to process rank 4 with PID 21502 on
node beda63-3.nfs.private exiting without calling "finalize". This may
have caused other processes in the application to be
terminated by signals sent by mpirun (as reported here).
--------------------------------------------------------------------------
[beda63-3.nfs.private:21497] 7 more processes have sent help message help-mpi-api.txt / mpi-abort
[beda63-3.nfs.private:21497] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
